libsentrykube/kube.py

```diff
@@ -120,6 +120,20 @@ def _consolidate_variables(
        preserve comments.
     6. overridden by the cluster file. Which is likely going to be replaced by 2 and 3.
 
+    ***
+    For all levels of overrides listed above, in the same directory of file system, we support
+    merging separete values files together, before proceding with the overriding logic
+    Most common examples (numbers refer to override level listed above):
+    1. `k8s/services/getsentry/_values.yaml` content will be combined with `k8s/services/getsentry/_values_consumers.yaml`
+    3. `k8s/services/getsentry/regional_overrides/s4s/_values.yaml` content will be combined with
+       `k8s/services/getsentry/regional_overrides/s4s/_values_consumers.yaml`
+    4. `sk8s/services/seer/region_overrides/de/default.yaml` content will be combined with ``sk8s/services/seer/region_overrides/de/default_2.yaml`
```

Comment:
typos
```suggestion
    4. `k8s/services/seer/region_overrides/de/default.yaml` content will be combined with `k8s/services/seer/region_overrides/de/default_2.yaml`
```

Is there a reason we use a `default_2` example instead of a `default_consumers`?

Reply:
just because i don't think the `seer` service would have a consumers file, i was trying to give real examples. Not sure what example to give here.